### ISSUE TITLE
Page or Post Editor > post-title-wrapper > enhance max width (UX / responsiveness)

### ISSUE THREAD

**Issue by Marc-pi** [WordPress/gutenberg#53724]
### Description

By default, a max-width is set to 650px, so few words are displayed on several lines, which lower the available space and impacts the writing flow (we have to scroll more).  On desktop, we have far more available space

![image](https://github.com/WordPress/gutenberg/assets/919332/556070a7-56e7-40b8-bc43-ce807fcd7f3d)

It is something that is relative to responsiveness. The 650px is too hard.
Can we improve this, for instance, by 
- setting the max-with to something higher, 
- and maybe by lowering the font size a little bit
- and maybe encapsulate this title more design wise since it is not purely within the template

Actually it generates a bad experience. The novice user does not understand why this is displayed on several lines, but does not know it is only "the post name", which is not always displayed in front (depending on the template)

ping @richtabor > since it seems to be a "quickwin" and you are sensitive to UX things :-)

### Step-by-step reproduction instructions

1. Just add a page or post with a quite short name
2. Edit the page, and observe how it is displayed

### Screenshots, screen recording, code snippet

```


.editor-styles-wrapper .edit-post-visual-editor__post-title-wrapper > :where(:not(.alignleft):not(.alignright):not(.alignfull)) {
  max-width: 650px;
  margin-left: auto !important;
  margin-right: auto !important;
}

```

### Environment info

- WP6.2/6.3 and below
- TT2/TT3, but here it deals with admin UX when editing a post/page

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by hanneslsm** [comment]
Not sure if I fully understood the issue.
The default content width can be adjusted in Global Styles → Layout → Content. Is this what you were looking for?

**Comment by Marc-pi** [comment]
**on backoffice** (not on front), when editing your posts or pages, the title wraper space is too short on desktop.
​it is nonsense to reduce the available space like this, it brings a bad experience when using the Gutenberg Editor
Let's adjust so new Gutenberg users have a good experience at first sight !!

**Comment by hanneslsm** [comment]
> on backoffice (not on front)

Changing the content width in global style should also change it in the editor.

> ​it is nonsense to reduce the available space like this

The available space is related to the screen resolution. On my MacBook Pro for example, the is not much more space available space (Content width set to 620px)

<img width="1511" alt="Screenshot 2023-08-16 at 13 58 02" src="https://github.com/WordPress/gutenberg/assets/17861443/9192b95e-72d3-409c-bc30-a1eb2e413c3d">



> the title wraper space is too short on desktop

Are you specifically talking about the space in the title wrapper? You can change this by adding this to your edior-style.css:

```
.edit-post-visual-editor__post-title-wrapper h1 {
  max-width: 100vw;
}
```

**Comment by Marc-pi** [comment]
> Are you specifically talking about the space in the title wrapper

yes, and since it is just an internal title (it is just to be displayed on backoffice or used for Title html tag  > you do not see it in the View List), it makes no sense to adjust its size based on theme sizes.

IMHO it is something to change, and we need to design it better so user understands it is just the (internal) name of the page
Coz it is not part of the template Block View List. 

Maybe with a gray background, to differenciate it. 
There is here an UX issue for any new user onboarding on WP Gutenberg. 

poke @richtabor / @jasmussen

**Comment by hanneslsm** [comment]
> we need to design it better so user understands it is just the (internal) name of the page
> Coz it is not part of the template Block View List.

I agree with you here and I solve it by[ visually differentiating the title in my themes. ](https://github.com/hanneslsm/ProLooksUI/blob/8856a7b1ac7c1df18a8e659e802f9158a43238e8/editor-style.css#L4C40-L4C40)

**Comment by jasmussen** [comment]
If I'm reading this right, apologies if not, then this is an issue mainly with themes that do not include layout properties for content widths in a theme.json file, correct? If yes, then it should be possible to add a theme.json and add your custom value there. [There should be details here](https://developer.wordpress.org/themes/advanced-topics/theme-json/#layout).

**Comment by ryanAllMad** [comment]
Sorry if this isn't on topic, but I believe it is. The issue I'm seeing that relates to this is that the width of the post title block from the site editor isn't respected in the post editor.

In 2023, set the post title block to align wide in the single template:
![Screenshot 2023-09-21 at 10 31 52 AM](https://github.com/WordPress/gutenberg/assets/46852695/22b03a3c-78ba-4048-be2d-4b946d988cde)

In the Post Editor on a single template, note the width of the post title block is still the contentSize not the wideSize:
![Screenshot 2023-09-21 at 10 32 41 AM](https://github.com/WordPress/gutenberg/assets/46852695/87b28583-a4b5-4f5f-93d0-cd3f1b674708)

Note that I can alter the post content block width in the template editor:
![Screenshot 2023-09-21 at 10 35 45 AM](https://github.com/WordPress/gutenberg/assets/46852695/ed850045-c6ae-440f-aa70-879553dde755)

And it is respected in the Post Editor:
![Screenshot 2023-09-21 at 10 36 00 AM](https://github.com/WordPress/gutenberg/assets/46852695/8e2e11f9-859f-47fa-92e1-ed71a60faded)
